### ISSUE TITLE
[104] false positive

### ISSUE THREAD

**Issue by MarkusTeufelberger** [ansible-community/ansible-lint#508]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible-lint --version
ansible-lint 4.1.0
```

Reproducing playbook:

```
---
- hosts: all
  tasks:
    - name: foo
      set_fact:
        bar: "{{ item }}"
      with_items: "{{ foo.split('\n') }}"
```

# Desired Behaviour

Error 104 doesn't trigger.

# Actual Behaviour (Bug report only)

```
foo.yml:4: [E104] Found a bare variable '{{ foo.split('
') }}' used in a 'with_items' loop. You should use the full variable syntax ('{{{{ foo.split('
') }}}}')
```

**Comment by ssbarnea** [comment]
Not reproducible with code from master.
